### ISSUE TITLE
Visualization

### ISSUE THREAD

**Issue by alfredgu001324** [Mrmoore98/VectorMapNet_code#39]
Hi author, thanks for posting the code. I noticed that there is a visualization folder, but is there any specific instructions about how to run it? As 'data' appears to have three keys only ['img_metas', 'img', 'polys'] and the rendering would not work.

Really appreciate for your help and looking forward to your reply!

**Comment by QixingJiang** [comment]
+1

**Comment by aidadgy** [comment]
> Hi author, thanks for posting the code. I noticed that there is a visualization folder, but is there any specific instructions about how to run it? As 'data' appears to have three keys only ['img_metas', 'img', 'polys'] and the rendering would not work.
> 
> Really appreciate for your help and looking forward to your reply!

Hello, have you solved this problem? How to achieve visualization

**Comment by QixingJiang** [comment]
> > 嗨，作者，感谢您发布代码。我注意到有一个可视化文件夹，但是有没有关于如何运行它的具体说明？由于“数据”似乎只有三个键 ['img_metas'、'img'、'polys']，因此渲染不起作用。
> > 非常感谢您的帮助并期待您的回复！
> 
> 你好，这个问题解决了吗？如何实现可视化

Hi，I have failed in visuallizing the results, and I turned to HDMapNet and MapTR, which can easily visualize the prediction. I think the problem of VectorMapNet remains unsolved unless the code is updated.

**Comment by aidadgy** [comment]
> > > 嗨，作者，感谢您发布代码。我注意到有一个可视化文件夹，但是有没有关于如何运行它的具体说明？由于“数据”似乎只有三个键 ['img_metas'、'img'、'polys']，因此渲染不起作用。
> > > 非常感谢您的帮助并期待您的回复！
> > 
> > 
> > 你好，这个问题解决了吗？如何实现可视化
> 
> Hi，I have failed in visuallizing the results, and I turned to HDMapNet and MapTR, which can easily visualize the prediction. I think the problem of VectorMapNet remains unsolved unless the code is updated.
Okay, but MapTR testing requires 8GPUS, and I only have a single GPU

**Comment by HU75** [comment]
@QixingJiang Hi, Do you mean to display the results of VectorMapNet by using the visualization scripts of HDMapNet or MapTR?

**Comment by QixingJiang** [comment]
> [@QixingJiang](https://github.com/QixingJiang)你好，你的意思是用HDMapNet或者MapTR的可视化脚本来展示VectorMapNet的结果吗？

No, I meaned that I turn to MapTR and HDMapNet to visualize MapTR and HDMapNet, i.e., their results of themselves
